cli/src/test/java/com/devonfw/tools/ide/tool/intellij/IntellijTest.java

```diff
@@ -17,20 +20,28 @@
 import com.devonfw.tools.ide.log.IdeLogLevel;
 import com.devonfw.tools.ide.os.SystemInfo;
 import com.devonfw.tools.ide.os.SystemInfoMock;
+import com.github.tomakehurst.wiremock.junit5.WireMockRuntimeInfo;
 import com.github.tomakehurst.wiremock.junit5.WireMockTest;
 
 /**
  * Integration test of {@link Intellij}.
  */
-// TODO: replace with dynamic ports, see: https://github.com/devonfw/IDEasy/pull/487
-@WireMockTest(httpPort = 8080)
+@WireMockTest
 public class IntellijTest extends AbstractIdeContextTest {
 
   private static final String PROJECT_INTELLIJ = "intellij";
   private static final String MOCKED_PLUGIN_JAR = "mocked-plugin.jar";
+  private static final String TEST_DATA_ROOT = "src/test/resources/ide-projects/intellij/project/settings/intellij/plugins";
 
   private final IdeTestContext context = newContext(PROJECT_INTELLIJ);
 
+
+  @BeforeAll
+  public static void setup(WireMockRuntimeInfo wmRuntimeInfo) throws IOException {
+    String content = "plugin_id=mockedPlugin\nplugin_active=true\nplugin_url=" + wmRuntimeInfo.getHttpBaseUrl() + "/mockedPlugin";
+    Files.write(Path.of(TEST_DATA_ROOT).resolve("MockedPlugin.properties"), content.getBytes(StandardCharsets.UTF_8));
```

Comment:
a golden rule for JUnits is that they should never do mutations that would be tracked by git.
Wouldn't it be possible and even easier to simply write the file in target relative to `context` in `setupMockedPlugin()`?